include/zenoh-pico/link/link.h

```diff
@@ -46,22 +46,18 @@
  * Link capabilities values, defined as a bitmask.
  *
  * Enumerators:
- *     Z_LINK_CAPABILITY_NONE: Bitmask to define that link has no capabilities.
- *     Z_LINK_CAPABILITY_RELIABLE: Bitmask to define and check if link is reliable.
- *     Z_LINK_CAPABILITY_STREAMED: Bitmask to define and check if link is streamed.
- *     Z_LINK_CAPABILITY_MULTICAST: Bitmask to define and check if link is multicast.
+ *     Z_LINK_CAP_UNICAST_STREAM: Link has unicast stream capabilities.
+ *     Z_LINK_CAP_UNICAST_DATAGRAM: Link has unicast datagram capabilities.
+ *     Z_LINK_CAP_MULTICAST_STREAM: Link has multicast stream capabilities.
+ *     Z_LINK_CAP_MULTICAST_DATAGRAM: Link has multicast datagram capabilities.
  */
 typedef enum {
-    Z_LINK_CAPABILITY_NONE = 0x00,      // 0
-    Z_LINK_CAPABILITY_RELIABLE = 0x01,  // 1 << 0
-    Z_LINK_CAPABILITY_STREAMED = 0x02,  // 1 << 1
-    Z_LINK_CAPABILITY_MULTICAST = 0x04  // 1 << 2
+    Z_LINK_CAP_UNICAST_STREAM = 0,
+    Z_LINK_CAP_UNICAST_DATAGRAM,
+    Z_LINK_CAP_MULTICAST_STREAM,
+    Z_LINK_CAP_MULTICAST_DATAGRAM,
```

Comment:
I would opt to always have the enumeration values set, as a best practice.
We had some misalignments in the past because of this, due to changing the order which reassigned different values.